express/routes/productsRouter.js

```diff
@@ -0,0 +1,22 @@
+import express from "express";
+
+const router = express.Router();
+
+router
+	.route("/")
+	.get((req, res) => {
```

Comment:
Please, extract route middleware to `controllers` folder in files named like `products.controller.js`